Docs: adding editor types to setup docs. (#16514)
ckeditor/ckeditor5: packages/ckeditor5-ui/docs/framework/external-ui.md

## Readying the editor side

> The editor types such as  {@link getting-started/setup/editor-types#classic-editor classic editor} or {@link getting-started/setup/editor-types#inline-editor inline editor} come with a dedicated default user interface and a theme. However, to create an editor instance bound to a Bootstrap UI, only a limited subset of features is required. You need to import them first:

Reviewer: ```suggestion
The editor types, such as the {@link getting-started/setup/editor-types#classic-editor classic} or {@link getting-started/setup/editor-types#inline-editor inline editor}, have a dedicated default user interface and a theme. However, only a limited subset of features is required to create an editor instance bound to a Bootstrap UI. You need to import them first:
```
